test/integration/dryrun/dryrun_test.go

```diff
@@ -241,6 +247,14 @@ func TestDryRun(t *testing.T) {
 
 	dryrunData := etcd.GetEtcdStorageData()
 
+	scaleRestMapping := map[string]interface{}{
+		"ReplicationController": &repContStorage.ScaleREST{},
+		"ReplicaSet":            &repSetStorage.ScaleREST{},
+		"Deployment":            &deployStorage.ScaleREST{},
+		"StatefulSet":           &statSetStorage.ScaleREST{},
+		"DaemonSet":             &daemonStorage.ScaleREST{},
+	}
```

Comment:
linking to REST implementations in integration tests is pretty strange... if this is trying to see if the scale subresource supports `update`, that information is available in discovery (`/apis/apps/v1` would contain a `daemonsets/scale` entry with `verbs` that does not contain `update`, etc)